File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/Currencies/ICurrencyData.cs
using Ohtap.ECommerce.Domain.Models.Objects.Datasets;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Currencies
{
    public interface ICurrencyData : CoreObject.IObjectID, CoreObject.IObjectCode, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, ICurrencySymbol, ICurrencyFundCode
    {
        IDataItemList<IDataItemData> Dataset { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Currencies/ICurrencyData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Currencies;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface ICurrencyData extends ICurrencyFundCode,ICurrencySymbol,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
